client/components/editor/structure/TesList.vue

```diff
@@ -7,7 +7,7 @@
       @end="dragElementIndex = -1"
       @update="$emit('update', $event)"
       :list="list"
-      :options="options"
+      v-bind="options"
```

Comment:
The `options` computed is created by assigning 3 properties to the `dragOptions` component prop. For one, we shouldn't be assigning anything to a component prop so at least that needs to change. And second, we never pass any properties to this component. It always defaults. So my question is for @vladimyr  and @kronicker , do we want to leave this option to pass the prop or just remove it altogether?

Reply:
Lets leave it as is for now (for sake of getting this merged) but that should be addressed later 👍